string current_pose
bool moving
string[] scene_objects
string[] attached_objects
Command got_command
